diff --git a/nmake.opt b/nmake.opt
index a29fdb7..e43d0ad 100644
--- a/nmake.opt
+++ b/nmake.opt
@@ -307,9 +307,9 @@ JPEG_SUPPORTED = 1
 # This will enable 12bit libjpeg, using internal IJG libjpeg-6b in 12-bit mode
 # Note: it is possible to have 12bit JPEG support with this internal IJG libjpeg-6b
 # and 8bit JPEG support through an external libjpeg, such as libjpeg-turbo
-!IFNDEF JPEG12_SUPPORTED
-JPEG12_SUPPORTED = 1
-!ENDIF
+#!IFNDEF JPEG12_SUPPORTED
+#JPEG12_SUPPORTED = 1
+#!ENDIF
 
 #if using an external jpeg library uncomment the following lines
 # libjpeg-turbo is also supported (and recommended for better performance)
