diff --git a/frmts/hdf5/makefile.vc b/frmts/hdf5/makefile.vc
index ca3081e..217a883 100644
--- a/frmts/hdf5/makefile.vc
+++ b/frmts/hdf5/makefile.vc
@@ -8,7 +8,7 @@ OBJ		=	hdf5dataset.obj  hdf5imagedataset.obj \
 
 PLUGIN_DLL 	=	gdal_HDF5.dll
 
-EXTRAFLAGS = -I$(HDF5_DIR)\include -DWIN32 
+EXTRAFLAGS = -I$(HDF5_DIR)\include -DWIN32 -DH5_BUILT_AS_DYNAMIC_LIB
 
 # From the HDF docs...
 # Users who want to build and run an application with HDF5 in Visual Studio
